distributions/jamulus-server.service

```
@@ -8,7 +8,7 @@ Type=simple
 Restart=always
 RestartSec=1
 User=jamulus
-ExecStart=/bin/sh -c '/usr/bin/jamulus -s -n -l /var/log/jamulus -e jamulus.fischvolk.de -g -o "$(uname -n);;"'
+ExecStart=/usr/bin/jamulus -s -n -l /var/log/jamulus -e anygenre1.jamulus.io:22124 -g -o "$(uname -n);;"
```

I suspect `$(uname -n)` will not expand properly without the `sh -c` wrap. This would likely need the `exec` as @softins suggested (haven't tried myself):

```suggestion
ExecStart=sh -c 'exec /usr/bin/jamulus -s -n -l /var/log/jamulus -e anygenre1.jamulus.io:22124 -g -o "$(uname -n);;"'
```

---

I would expect it to be OK since it's not a `sh` only thing. Putting `sh -c` back here will cause the problem with SIGUSR2, no?

---

I wasn't aware of the second file. @hoffie suggested change will work. I already tested it with SIGUSR2.

---

If I read https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers correctly

    ExecStart=/usr/bin/jamulus -s -n -l /var/log/jamulus -e anygenre1.jamulus.io:22124 -g -o %H

would work.